client/modules/food/components/packing/Packages.js

```diff
@@ -8,27 +8,41 @@ import {BootstrapTable, TableHeaderColumn, SizePerPageDropDown} from 'react-boot
 import 'react-bootstrap-table/dist/react-bootstrap-table.min.css'
 
 import selectors from '../../../../store/selectors'
-import {listPackages, unpackPackage} from '../../reducers/packing'
+import {listPackages, unpackPackage, receivedPackage} from '../../reducers/packing'
```

Comment:
As the food bank, the packages are delivered not received, I'd change the names in the code and ui text